### ISSUE TITLE
Plans to make for classic?

### ISSUE THREAD

**Issue by firefang03** [shirsig/CDFrames#35]

**Comment by Sneakyrhino1** [comment]
this would be insane,pls

**Comment by stage1st** [comment]
+1

**Comment by shirsig** [comment]
No. Try this one maybe https://www.curseforge.com/wow/addons/nameplatecooldowns/files

**Comment by heydanbrowne** [comment]
@shirsig mate you're cdframes worked so much differently and generally better than that nameplatecooldowns addon. please save us and update this if you can - this seriously is the best pvp addon from vanilla pservers

**Comment by danielbjornadal** [comment]
> No. Try this one maybe https://www.curseforge.com/wow/addons/nameplatecooldowns/files

They now have restricted nameplates to 20y. 
The cdframes-addon is king. If you are not going to port it to classic, could you atleast give us a pointer to what needs to be changed? 
Like parsing of combat logs, or other adjustments.

**Comment by shirsig** [comment]
It would probably be easier to make a different gui for that other addon than to port this. The classic API is very close to retail and not close at all to vanilla.

**Comment by mefire66** [comment]
I am dying to have this addon in classic.
